cypress/integration/features/task/createEditAndDeleteTask.feature

```diff
@@ -38,15 +40,90 @@ Scenario: Teacher edits and publishes task from room via form
     And I click on button Submit
     Then I can see room page 'Course with subject and tasks'
     And I can see task 'Cy Task Creating, Editing, Deleting Test'
+    And Task card info submitted contains "0/2" for task 'Cy Task Creating, Editing, Deleting Test'
     When I click on task 'Cy Task Creating, Editing, Deleting Test'
     Then description is 'Dies ist Deine Aufgabe. Viel Erfolg!'
-    When I click on button Edit
+    And file 'example_jpg.jpg' is visible in section files
+    When I click on file 'example_jpg.jpg'
+    Then file 'example_jpg.jpg' is displayed in file-viewer
+    When I click on file-viewer for file 'example_jpg.jpg'
+    And I click on button Edit
     Then Public Submission is enabled
```

Comment:
--> Then I see public submission is enabled

Reply:
done